react/PieChart/index.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import { Chart as ChartJS, ArcElement, Tooltip } from 'chart.js'
+import { Doughnut } from 'react-chartjs-2'
+import { useTheme } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+import set from 'lodash/set'
+import cloneDeep from 'lodash/cloneDeep'
+
+import Typography from '../Typography'
+import isTesting from '../helpers/isTesting'
+
+ChartJS.register(ArcElement, Tooltip)
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    position: 'relative',
+    zIndex: '1',
+    width: '192px',
+    height: '192px'
+  },
+  background: {
+    position: 'absolute',
+    boxSizing: 'border-box',
+    top: 0,
+    bottom: 0,
+    left: 0,
+    right: 0,
+    border: '24px solid var(--actionColorGhost)',
+    borderRadius: '100%',
+    zIndex: -1,
+    backgroundColor: ({ single }) =>
+      !single && theme.palette.type === 'dark'
+        ? theme.palette.primary.main
+        : 'none',
+    boxShadow: ({ single }) =>
+      !single && theme.palette.type === 'dark'
+        ? `0 0 0 2px ${theme.palette.primary.main}`
+        : 'none'
+  },
+  centerText: {
+    display: 'flex',
+    flexDirection: 'column',
+    position: 'absolute',
+    alignItems: 'center',
+    justifyContent: 'center',
+    textAlign: 'center',
+    top: '25px',
+    left: '25px',
+    right: '25px',
+    bottom: '25px',
+    borderRadius: '50%',
+    zIndex: '-1'
+  },
+  typography: {
+    color: ({ single }) =>
+      !single && theme.palette.type === 'dark'
+        ? theme.palette.primary.contrastText
+        : theme.palette.text.primary
+  }
+}))
+
+const makeOptions = (options, single) => {
+  const madeOptions = {
+    cutout: '75%',
+    elements: {
+      arc: {
+        borderWidth: 0
+      }
+    },
+    animation: isTesting() ? false : { duration: 1000 },
```

Comment:
instead of setting isTesting method inside component that will be used outside of cozy-ui, why not providing a prop that would disable animation?

Reply:
parce qu'on ne veut pas se comportement possible depuis l'app, le besoin est spécifique à la doc cozy-ui

Reply:
un peu ennuyant d'ajouter ça dans le code utilisé par les autres repos. mais ok